flink-libraries/flink-gelly-examples/src/main/java/org/apache/flink/graph/examples/data/TriangleCountData.java

```diff
@@ -33,7 +33,15 @@
  */
 public class TriangleCountData {
 
-	public static final String EDGES = "1	2\n"+"1	3\n"+"2	3\n"+"2	6\n"+"3	4\n"+"3	5\n"+"3	6\n"+"4	5\n"+"6	7\n";
```

Comment:
No, there just happen to be 7 characters between tabs so they appear as spaces after the first.